### PR TITLE
PMM-5645 Update go-swagger to use go 1.14

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-5645

refs:
 - https://github.com/percona/pmm/pull/540
 - https://github.com/Percona-Lab/spec/pull/1
 - percona/pmm-admin#114
 - percona/pmm-agent#173
 - percona/qan-api2#91
 - percona/pmm-managed#413
 - Percona-Lab/pmm-api-tests#110

Left: current FB, Right: 2.7.0

image

image

image

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-ba89a3f
client docker - perconalab/pmm-client-fb:PR-910-ba89a3f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-ba89a3f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-a4cdbfe
client docker - perconalab/pmm-client-fb:PR-910-a4cdbfe
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-a4cdbfe.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-329a47a
client docker - perconalab/pmm-client-fb:PR-910-329a47a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-329a47a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-1da2178
client docker - perconalab/pmm-client-fb:PR-910-1da2178
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-1da2178.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-7f24990
client docker - perconalab/pmm-client-fb:PR-910-7f24990
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-7f24990.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-1f6d0be
client docker - perconalab/pmm-client-fb:PR-910-1f6d0be
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-1f6d0be.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-17abe31
client docker - perconalab/pmm-client-fb:PR-910-17abe31
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-17abe31.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-bde67e0
client docker - perconalab/pmm-client-fb:PR-910-bde67e0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-bde67e0.tar.gz

File: build/rpmbuild-docker/Dockerfile [hunk 1]
```diff
@@ -21,8 +21,8 @@ RUN yum install -y --setopt=skip_missing_names_on_install=False \
 RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
-ENV GO_VERSION 1.13.10
-ENV GO_CHECKSUM 8a4cbc9f2b95d114c38f6cbe94a45372d48c604b707db2057c787398dfbf8e7f
+ENV GO_VERSION 1.14.4
+ENV GO_CHECKSUM aed845e4185a0b2a3c3d5e1d0a35491702c55889192bb9c30e67a3de6849c067
```
[the remark this answers is not in the document]
Comment (askomorokhov) [reply build/rpmbuild-docker/Dockerfile]: Added separate PR https://github.com/Percona-Lab/pmm-submodules/pull/918

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-84633b4
client docker - perconalab/pmm-client-fb:PR-910-84633b4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-84633b4.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-910-84633b4
client docker - perconalab/pmm-client-fb:PR-910-84633b4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-910-84633b4.tar.gz

**Pull request closed without merging**